Assets/Scripts/GameManager.cs

```diff
@@ -19,12 +20,20 @@ void Start()
         numCoins = 0;
         isGameOver = false;
         isFinished = false;
+        isGameCompleted = false;
     }
 
     // Update is called once per frame
     void Update()
     {
-        
+        if (isGameOver)
+        {
+            // Restart game if player presses R
+            if (Input.GetKeyDown(KeyCode.R))
```

Comment:
Does the player know if they have to press letter R for restart?

Reply:
No